mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWriteTagActivity.java

```diff
@@ -187,8 +179,7 @@ public void writeTag(Tag tag, String openhabUri) {
                 writeTagMessage.setText(R.string.info_write_tag_finished);
                 autoCloseActivity();
             } catch (IOException | FormatException e) {
-                if (e.getMessage() != null)
-                    Log.e(TAG, e.getMessage());
+                Log.e(TAG, "Writing to unformatted tag failed: " + e);
```

Comment:
`Log.e(TAG, "Writing to unformatted tag failed", e);`?

Reply:
I don't think the stack trace is actually useful here; that's why I chose this way.